js/test/Glacier2/router/Client.js

```diff
@@ -2,335 +2,327 @@
 // Copyright (c) ZeroC, Inc. All rights reserved.
 //
 
-(function(module, require, exports)
```

Comment:
For this tests an others, I just remove the wrapping IIFE, they are now Common JS modules and an follow up PR would convert them to ES6 modules.